plugins/cmdmenu.sma

```diff
@@ -140,26 +146,37 @@ public plugin_precache( )
 			copy( szSound, charsmax(szSound), field2 );
 			copy( sndExt, charsmax(sndExt), ".wav" );
 		}
-		else if ( fieldNums == 3 && field1[0] == 'm' && ( field2[0] == 'p' || field2[0] == 'l' ) )	// .mp3 (mp3 play | mp3 loop)
-		{
-			copy( szSound, charsmax(szSound), field3 );
-			copy( sndExt, charsmax(sndExt), ".mp3" );
+		else 
+		{	
```

Comment:
Why did you "expand" this condition? It was fine before. No reason to change it.